src/haz3lcore/statics/Translation.re

```diff
@@ -0,0 +1,402 @@
+let rec get_roc_exp = (t: TermBase.UExp.t, m: Statics.map): TermRoc.UExp.t =>
+  switch (t.term) {
+  | Invalid(_) => String("Invalid Not Implemented")
```

Comment:
instead of emitting a String here, define an `UnsupportedInput` exception and raise it in these cases